packages/ibm-products/src/components/EmptyStates/EmptyState.tsx

```diff
@@ -44,7 +44,7 @@ interface EmptyStateProps {
   action?: {
     kind?: 'primary' | 'secondary' | 'tertiary';
     renderIcon?: CarbonIconType;
-    onClick?: ButtonProps['onClick'];
+    onClick?: ButtonProps<'button'>['onClick'];
```

Comment:
For consistency, can you use `React.ElementType`? This is use elsewhere in the PR.